js/functions.js

```diff
@@ -1,18 +1,60 @@
+//array to check the operations.
+const arrayOfOperations = [
+  "1",
+  "2",
+  "3",
+  "4",
+  "5",
+  "6",
+  "7",
+  "8",
+  "9",
+  "0",
+  ".",
+  "+",
+  "-",
+  "/",
+  "*",
+  "**",
+  "!",
+  "**2",
+  "**3",
+  "%",
+  "rt",
+  "(",
+  ")",
+  "e",
+  "π",
+  "log1",
+  "log2",
+  "ln",
+  "e**",
+];
+
+//object to track the unit that changes on click
+let unitOfAngle = {
+  degree: true,
+  radian: false,
+  grad: false,
+};
+
+export const inputField = document.querySelector("textarea");
+
 //basic operation is present or not
-function isOperationPresent(clickedItem) {
+export function isOperationPresent(clickedItem) {
   return arrayOfOperations.includes(clickedItem) ? clickedItem : "#";
 }
 
 //fn that is going to just add the operation in the string
-function simpleCalculation(value) {
+export function simpleCalculation(value) {
   let string = getValueFromLocal("calString");
   if (string === undefined) string = "";
```

Comment:
We can check for falsy value like this `!string`.
Or if you are trying to guard for `null`/`undefined`, loose equality is very useful in such cases.

`string === undefined` checks only for `undefined`.
`string == null` checks for both.

Reply:
added the loose equality to enable both the checks.